html/semantics/interactive-elements/the-dialog-element/backdrop-descendant-selector-expected.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Generally, I think reference tests should be named `-ref.html`, rather than Chromium/WebKit's old `-expected.html` naming. I'm not sure that's mandatory, but it is certainly customary.